### DISCUSSION TITLE
Uuups! Unexpected error... | Cannot load the web interface

### DISCUSSION THREAD

**Discussion by Skyfay** [cockpit-project/cockpit-machines#663] [General]
I had version 215 in which everything worked. Now I've upgraded to the latest one and I keep getting this error.
![image](https://user-images.githubusercontent.com/76796007/162572808-1ebb8df2-3b57-4582-b7d2-c2f38071a2c0.png)
I also reinstalled it directly to this version once, but I got the same error.
![image](https://user-images.githubusercontent.com/76796007/162572853-62e340d3-4eee-44e3-ad8e-2fb9ffb7a569.png)
Can anybody help me further?

**Comment by martinpitt** [comment]
You didn't give much information (OS, installed package versions), but my first guess is that you upgraded some basic cockpit packages from Ubuntu backports, but did not upgrade cockpit-machines? Please try

    sudo apt install -t focal-backports cockpit-machines

**Reply by Skyfay** [reply]
Many thanks for the answer. In fact, I've issued this command before during my search.
But now it worked, everything is running again. Thanks very much.
